### PR TITLE
[tw] Add content/zh-tw/continuous-integration.md

### LINKED ISSUE #2101: [tw] Localize `Continuous integration (CI)` for Traditional Chinese
### Please describe why this term needs to be updated

- Localize `Continuous integration (CI)` in Traditional Chinese
- https://glossary.cncf.io/continuous-integration/

### PR DESCRIPTION
**Pull request by tico88612**
### Describe your changes

Origin content from https://glossary.cncf.io/continuous-integration/

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2101 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/continuous-integration.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+status: Completed 
+category: 概念
+tags: ["應用程式", "方法論", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+持續整合，通常縮寫為 CI，是盡可能定期整合程式碼更改的實踐方法。
+CI 是 [持續交付](/zh-tw/continuous-delivery/)（CD）的前置工作。
+傳統上，CI 的流程從程式碼更動提交到版本控管（Git、Mercurial 或 Subversion）時開始，以準備被 CD 系統使用的已測試組件結束。
```
Comment (ydFu) [inline content/zh-tw/continuous-integration.md:12]: original
```
Traditionally, the CI process begins when code changes are committed to a source control system (Git, Mercurial, or Subversion) 
and ends with a tested artifact ready to be consumed by a CD system. 
```

```suggestion
傳統上，CI 的流程從程式碼更動提交到版本控管（Git、Mercurial 或 Subversion）時開始，
以準備被 CD 系統使用的測試產出物（tested artifact）直到結束。
```

測試產出物（tested artifact）
1. [台灣與中國的計算機術語/用語對照表](https://hackmd.io/@SeanPeng/B1psY1s6K#%E5%8F%B0%E7%81%A3%E8%88%87%E4%B8%AD%E5%9C%8B%E7%9A%84%E8%A8%88%E7%AE%97%E6%A9%9F%E8%A1%93%E8%AA%9E%E7%94%A8%E8%AA%9E%E5%B0%8D%E7%85%A7%E8%A1%A8)
2.  [聊聊軟體交付的濫觴 談產出物管理 (Artifacts Management)](https://rickhw.github.io/2019/03/28/DevOps/DevOpsTaiwan-Meetup-Beginning-in-Artifacts-Management/)

File: content/zh-tw/continuous-integration.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## 解決的問題 {#problem-it-addresses}
+
+軟體系統通常是龐大且複雜的，有許多開發人員在維護和更新它們。
+在系統的不同部分同時工作，這些開發人員可能會做出互相衝突的修改，並在無意間破壞對方的工作。
```
Comment (ydFu) [inline content/zh-tw/continuous-integration.md:17]:
```suggestion
在系統的不同部分同時工作，
這些開發人員可能會做出互相衝突的修改，並在無意間破壞對方的工作。
```

File: content/zh-tw/continuous-integration.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+此外，由於多位開發人員在同一個專案上工作，任何日常任務，如測試和計算程式碼品質，都需要每位開發人員重複執行，浪費了時間。
```
Comment (ydFu) [inline content/zh-tw/continuous-integration.md:18]:
```suggestion
此外，由於多位開發人員在同一個專案上工作，
任何日常任務，如測試和計算程式碼品質，都需要每位開發人員重複執行，浪費了時間。
```

File: content/zh-tw/continuous-integration.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+## 如何幫助 {#how-it-helps}
+
+每當開發人員提交修改時，CI 軟體會自動檢查程式碼更動是否合併得很乾淨。
+使用 CI 伺服器來執行程式碼品質檢查、測試甚至部署，這幾乎是一種普遍的做法。
```
Comment (ydFu) [inline content/zh-tw/continuous-integration.md:23]:
```suggestion
使用 CI 伺服器來執行程式碼品質檢查、測試甚至部署，這幾乎是一種普遍存在的做法。
```
1. [劍橋辭典-ubiquitous ](
https://dictionary.cambridge.org/zht/%E8%A9%9E%E5%85%B8/%E8%8B%B1%E8%AA%9E-%E6%BC%A2%E8%AA%9E-%E7%B9%81%E9%AB%94/ubiquitous)

**Review by ydFu — COMMENTED** [review]
According to the style guide, one sentence per line is recommended, following the original.

https://glossary.cncf.io/style-guide/#three-subheadings

**Review by pichuang — APPROVED** [review]
LGTM

**Review by ydFu — APPROVED** [review]
LGTM, thank @tico88612

**Pull request merged**